temba/channels/types/whatsapp_cloud/type.py

```diff
@@ -120,3 +120,65 @@ def get_api_templates(self, channel):
         except requests.RequestException as e:
             HTTPLog.create_from_exception(HTTPLog.WHATSAPP_TEMPLATES_SYNCED, url, e, start, channel=channel)
             return [], False
+
+    def get_api_catalogs(self, channel):
+        if not settings.WHATSAPP_ADMIN_SYSTEM_USER_TOKEN:  # pragma: no cover
+            return [], False
+
+        waba_id = channel.config.get("wa_business_id", None)
+        if not waba_id:  # pragma: no cover
+            return [], False
+
+        start = timezone.now()
+        try:
+            catalog_data = []
+            url = f"https://graph.facebook.com/v16.0/{waba_id}/owned_product_catalogs"
```

Comment:
I suggest you create an issue to change the code snippets that use the Facebook API version statically, it can be defined in the settings and used when necessary at other points, if the version changes you would only change it in the settings.

Reply:
This kind of update should affect all the tasks related to Meta. I'll put it on backlog to do it later.